ts/Core/Tooltip.ts

```diff
@@ -177,7 +177,7 @@ declare global {
             plotX: number;
             plotY: number;
         }
-        type TooltipShapeValue = ('callout'|'circle'|'square');
+        type TooltipShapeValue = ('callout'|'circle'|'square'|'rect');
```

Comment:
As above - `tooltip.shape = 'square'` will actually render a rectangle, so I think it is better to remove `square`